internal/cmd/controller/reconciler/bundle_controller.go

```diff
@@ -335,6 +335,15 @@ func (r *BundleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		// Changes in the values hash trigger a bundle deployment reconcile.
 		bd.Spec.ValuesHash = valuesHash
 
+		// When content resources are stored in etcd, we need to keep track of the content resource so they
+		// are properly gargabe collected by the content controller.
```

Comment:
nit:
```suggestion
		// are properly garbage-collected by the content controller.
```

Reply:
I've searched at the whole project and I can't find it.

Reply:
will fix in a future PR, sorry for the confusion